### PR TITLE
run `install_prereqs` only once on a host

### PR DESCRIPTION
**Pull request by darkonie**
related PR: https://github.com/dcos/dcos/pull/477

### CODE REVIEW

File: ext/dcos-installer/dcos_installer/action_lib/__init__.py [hunk 1]
```diff
@@ -322,6 +323,11 @@ def _add_prereqs_script(chain):
   exit 0
 fi
 
+if [ -f /opt/prereqs.installed ]; then
```
Comment (cmaloney) [inline ext/dcos-installer/dcos_installer/action_lib/__init__.py]: Can we add `dcos` to the name of this somehow / somewhere? (`/opt/dcos-prereqs.installed`)

**Comment by cmaloney** [comment]
Seems reasonable to me as an MVP fix

**Comment by darkonie** [comment]
@cmaloney fixed, using `/opt/dcos-prereqs.installed` flag

**Comment by spahl** [comment]
:ship:

**Pull request merged**